src/assets/persons.js

```diff
@@ -15202,22 +15202,6 @@ export const data = {
         country: "India",
       },
     },
-    {
-      id: uuidv4(),
```

Comment:
This pull request will remove Akshay from the person.js file. Please address this

Reply:
I am intentionally removing it, because its a duplicate. The same data exists around line 20730.